### PR TITLE
Update release notes for 2023.12.17 r1

### PR DESCRIPTION
**Pull request by cpaulgilman**
Release notes updates for SAM 2023.12.17 r1.

Merge into gh-pages branch, not patch or develop.

Wait to merge this until release files are posted.

**DO NOT MERGE INTO PATCH OR DEVELOP**

### CODE REVIEW

File: doc/releasenotes.html [hunk 1]
```diff
@@ -59,6 +59,55 @@ <h2>CATEGORY: General, Solar, Detailed Photovoltaic, Wind Power, CSP Physical Tr
...
+<li>Fix Detailed PV non-linear self shading for vertical arrays to avoid negative POA irradiance values. (<a href="https://github.com/NREL/ssc/pull/1127">SSC 1127</a>)</li>
+<li>Fix issue with Detailed PV subhourly clipping loss when weather file has no DNI data. (<a href="https://github.com/NREL/ssc/pull/1126">SSC 1126</a>)</li>
+<li>Fix reporting of ground-reflected rear irradiance gains in loss diagram for Detailed PV model with bifacial modules. (<a href="https://github.com/NREL/ssc/pull/1125">SSC 1125</a>)</li>
+<li>Add snow slide coefficient to outputs for Detailed PV with snow model enabled. (<a href="https://github.com/NREL/ssc/pull/1128">SSC 1128</a>)</li>
+<li>Clean up PV subhourly clipping loss correction in user interface. (<a href="https://github.com/NREL/SAM/pull/1689">SAM 1689</a>)</li>
+<li>Fix diagrams on Detailed PV Shading and Layout page to show both portrait and landscape instead of duplicating portrait. (<a href="https://github.com/NREL/SAM/pull/1696">SAM 1696</a>)</li>
+</ul>
+</li>
+<li>Concentrating Solar Power (CSP)
```
Comment (brtietz) [inline doc/releasenotes.html:85]: Additional issue not tagged with patch that was merged: https://github.com/NREL/SAM/pull/1711

File: doc/releasenotes.html [hunk 2]
```diff
@@ -59,6 +59,55 @@ <h2>CATEGORY: General, Solar, Detailed Photovoltaic, Wind Power, CSP Physical Tr
 
 -->
 
+<h1 id="2023-12-17-r1">SAM 2023.12.17 r1, SSC 28x: March xx, 2024</h1>
+
+<p>This is the first maintenance update to <a href="#2023-12-17">SAM 2023.12.17</a>.</p>
+
+<ul>
+<li>Solar Resource
```
Comment (brtietz) [inline doc/releasenotes.html:67]: Additional solar resource issue that was fixed: https://github.com/NREL/SAM/pull/1710

File: doc/releasenotes.html [hunk 3]
```diff
@@ -59,6 +59,55 @@ <h2>CATEGORY: General, Solar, Detailed Photovoltaic, Wind Power, CSP Physical Tr
...
+<li>Fix simulation error caused by battery environment temperature input as time series array. (<a href="https://github.com/NREL/SAM/pull/1694">SAM 1694</a>, <a href="https://github.com/NREL/ssc/pull/1138">SSC 1138</a>)</li>
+<li>Fix issue with behind-the-meter battery with retail rate dispatch that caused a simulation error. (<a href="https://github.com/NREL/ssc/pull/1135">SSC 1135</a>)
+<li>Fix default battery life inputs so correct table is shown for the default Li-ion LFP/Graphite battery type.  (<a href="https://github.com/NREL/SAM/pull/1715">SAM 1715</a>)
+<li>Fix problem with battery discharging to grid for manual dispatch option. (<a href="https://github.com/NREL/ssc/pull/1115">SSC 1115</a>)</li>
+<li>Fix problem with battery self consumption dispatch option with grid outage. (<a href="https://github.com/NREL/ssc/pull/1114">SSC 1114</a>)</li>
+</ul>
+</li>
+<li>Fix problem with losses specifed as "time series losses with custom time periods" that caused a simulation error. (<a href="https://github.com/NREL/SAM/pull/1698">SAM 1698</a>)</li>
```
Comment (brtietz) [inline doc/releasenotes.html:101]: It looks like "financial models header" might be missing? (not sure, reading this in html)

https://github.com/NREL/ssc/pull/1129 should have been tagged with the patch as well.

**Review by brtietz — COMMENTED** [review]
Thanks for putting this together! I tagged a few additional pull requests with the patch in the comments where the author missed tagging.

Some additional items may need to be added as things are merged today (e.g. https://github.com/NREL/ssc/pull/1137)

**Comment by mjprilliman** [comment]
> [NREL/ssc#1137](https://github.com/NREL/ssc/pull/1137)

Just a reminder that https://github.com/NREL/ssc/pull/1137 is SDK only for now.

**Review by sjanzou — APPROVED** [review]
I am going to approve and merge for now and we can revisit after testing the patch 1 release candidate

**Pull request merged**